SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -105,11 +105,44 @@ void USpatialReceiver::LeaveCriticalSection()
 		{
 			OnEntityAddedDelegate.Broadcast(PendingAddEntity);
 		}
+		PendingAddComponents.RemoveAll([PendingAddEntity](const PendingAddComponentWrapper& Component) {return Component.EntityId == PendingAddEntity;});
```

Comment:
What's the purpose of this? Are these component adds already handled in some other way?

Reply:
Yeah, in ReceiveActor, we go through the relevant ones and handle them (but not remove them from the array, which is what this is doing).